user/useradm.go

```diff
@@ -423,6 +439,20 @@ func (ua *UserAdm) Verify(ctx context.Context, token *jwt.Token) error {
 		return errors.Wrap(err, "useradm: failed to get token")
 	}
 
+	// in case the token is a personal access token, update last used timestam
+	// to not overload the database with writes to tokens collection, we do not
+	// update the timestamp every time, but instead we wait some configurable
+	// amount of time between updates
+	if dbToken.TokenName != nil && ua.config.TokenLastUsedUpdateFreqMinutes > 0 {
+		t := time.Now().Add(
+			(-time.Minute * time.Duration(ua.config.TokenLastUsedUpdateFreqMinutes)))
```

Comment:
I would add a random jitter to this, the order fo seconds.

Reply:
not sure I understand

Reply:
treat as a suggestion: add a random number of seconds to the wait.